README.md

```diff
@@ -42,7 +42,7 @@ claims no expertise beyond English.
 | id     | Indonesian           | 4.0.0   | [ridho1991][id]      | Mutasim Ridlo
 | it     | Italian              | 4.0.0   | [luk3b8][it]         | Luca Rasia
 | ja     | Japanese             | 4.0.3   | [kenjis][ja]         | Kenji Suzuki
-| ko     | Korean               | 4.0.2   | [codeigniter-kr][kr] | codeigniter-kr
+| ko     | Korean               | 4.1.2   | [codeigniter-kr][kr] | codeigniter-kr
```

Comment:
Please revert this one. Version releases pertain to the version number where a locale is **first** introduced. So this should not change.

Reply:
I have just fixed it. :)